README.md

```diff
@@ -191,7 +191,6 @@ This is a list of modules currently in the repository (please add to the list if
 
  * katello_sync: sync Katello repositories and products
  * katello_upload: upload files, rpms, etc. to repositories. Note, rpms & files are idempotent.
- * katello_content_view_publish: publish Katello content views
```

Comment:
Can you add katello_content_view_version to the list above? I think it is missing.

Reply:
I actually considered dropping the list completely in favor of https://theforeman.org/plugins/foreman-ansible-modules

Reply:
... which is autogenerated. :1st_place_medal: 
For the riddance!

Reply:
I'd do that in a separate PR tho.

Reply:
Why? Do you want to collect PR-Points?